### PR TITLE
Password UI nits

### PR DESCRIPTION
**Pull request by ericawright**
smaller screens:
screen shot 2017-10-24 at 12 50 34 pm
screen shot 2017-10-24 at 12 50 07 pm

Tested on release versions of Chrome and Firefox

- Turns out HTML just doesn't remember spaces - so need to convert to `&nbsp;` to display passwords with spaces properly.

Fixes: #587.
Fixes: #610.
Fixes: #599.
Fixes: #591.
Fixes: #613.

### CODE REVIEW

File: public/locales/en-US/send.ftl [hunk 1]
```diff
@@ -85,5 +85,5 @@ footerLinkPrivacy = Privacy
 footerLinkTerms = Terms
 footerLinkCookies = Cookies
 requirePasswordCheckbox = Require a password to download this file
-addPasswordButton = Add Password
-incorrectPassword = Incorrect password. Try again?
+addPasswordButton = Add password
```
Comment (flodolo) [inline public/locales/en-US/send.ftl]: It's OK to change case without using a new ID.

File: public/locales/en-US/send.ftl [hunk 2]
```diff
@@ -85,5 +85,5 @@ footerLinkPrivacy = Privacy
...
+incorrectPassword = Incorrect password. Try again.
```
Comment (flodolo) [inline public/locales/en-US/send.ftl]: Unfortunately [this needs a new string ID](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings) if we want locales to update accordingly.

**Comment by sevaan** [comment]
Let's copy the formatting of the URL field, with the field on top and the button below.

File: public/locales/en-US/send.ftl (hunk 2 above)
Comment (ericawright) [reply public/locales/en-US/send.ftl]: thank you!

**Comment by ericawright** [comment]
@clouserw can you find a reviewer for this too?

**Comment by dannycoates** [comment]
This looks great, thanks @ericawright! 👍

**Pull request merged**